resources/views/help/user.twig

```diff
@@ -29,8 +29,9 @@ Before you can start translating, an admin has to grant you the necessary permis
 <p>By default, only languages that are being actively translated are listed. If yours isn't among them, it means that nobody has started a translation yet: Click <i>Show all languages</i> to see it.</p>
 
 <p>If the progress of a translation is below 100% the menu <i>Continue</i> beside the progress bar is highlighted blue and a click brings you to a page with all still unfinished text blocks.<br />
-In the menu, the item <i>All text blocks</i> opens that page with all text blocks, the ones already translated and the ones still needing work.<br />
-<i>Export</i> lets you download the catkeys of that particular language, just like <i>Export all</i> at the bottom of the list of languages lets you download the catkeys of all languages. The downloaded file can then be edited in a text editor or <a href="https://depot.haiku-os.org/#!/pkg/catkeyseditor/haikuports/0/1/2/-/3/x86_gcc2">CatKeysEditor</a>, if you prefer that.</p>
+In the menu, the item <i>All blocks</i> opens that page with all text blocks, the ones already translated and the ones still needing work.<br />
+<i>Pre-translate</i> opens a page with all the text blocks that have been automatically translated according to the <i>Glossaries<i> (see below). You should quickly move through that list and click on the <i>Save</i> button for all correct translations.<br /> 
```

Comment:
Two more things I think should be explicitly pointed out: checkboxes and textareas. Textareas allow to adjust the translation in case it is slightly wrong or just completely change it.